astropy/time/utils.py

```diff
@@ -49,6 +49,12 @@ def day_frac(val1, val2, factor=None, divisor=None):
     day = np.round(sum12)
     extra, frac = two_sum(sum12, -day)
     frac += extra + err12
+    # Our fraction can now have gotten >0.5 or <-0.5, which means we would
```

Comment:
It would be nice to include something in the docstring for `day_frac` that specifies the range `-0.5 <= frac <= 0.5` instead of just saying "fractional remainder".

Reply:
I adjusted the docstring (well, more like a rewrite...)